internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -470,6 +472,18 @@ func LoadConfig(seq runtime.Sequence, data interface{}) (runtime.TaskExecutionFu
 			if errors.Is(e, perrors.ErrNoConfigSource) {
 				logger.Println("machine configuration not found; starting maintenance service")
 
+				// nb: we treat maintenance mode as an "activate"
+				// event b/c the user is expected to be able to
+				// interact with the system at this point.
+				platform.FireEvent(
+					ctx,
+					r.State().Platform(),
+					platform.Event{
+						Type:    equinixmetal.EventTypeActivate,
```

Comment:
I feel these type constants should be in common `platform` package?

As it seems weird that common code imports equinixmetal specifically? 

We can for now pick EquinixMetal names for the constants, and add translation layer later as we have other platforms with event support.

Reply:
hrmm, yeah that seems like it makes sense. let me take a stab at that.